geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaCacheLoader.scala

```diff
@@ -95,6 +101,10 @@ object KafkaCacheLoader {
 
     private val serializer = new GeoMessageSerializer(sft, lazyDeserialization)
 
+    try { classOf[ConsumerRecord[Any, Any]].getMethod("timestamp") } catch {
```

Comment:
Would it be possible to handle this check *once* and capture the result as a lambda?  (Rather than having a try-catch inside the consume?

Reply:
it's only gets called once per simple feature type (per data store), as these consumers are cached in the data store. I could move it, but i figured it was probably good to log the message when you try to call .getSchema. What do you think?

Reply:
Once per featurestore is fine.